clients/snowflake/snowflake_test.go

```diff
@@ -156,6 +166,14 @@ func (s *SnowflakeTestSuite) TestExecuteMergeReestablishAuth() {
 }
 
 func (s *SnowflakeTestSuite) TestExecuteMerge() {
+	columnNames := []string{
```

Comment:
I added this because we were iterating over `colToKindDetailsMap` and ordering was not deterministic